.github/workflows/build.yml

```diff
@@ -0,0 +1,154 @@
+name: "Build"
+
+on:
+  pull_request:
```

Comment:
By default is not included `edited`, branch change does not trigger CI to run again. So there is a small chance to merge something with failing build.

```
  pull_request:
    types: [ "opened", "synchronize", "edited", "reopened" ]
```